hack/fuse-demo/coord-app.Dockerfile

```diff
@@ -46,5 +47,7 @@ RUN chmod a+x mock_application.sh
 
 USER developer
 
+RUN touch ~/.zshrc
+
 ENTRYPOINT [ "/tmp/coord/.scripts/wrap_application.sh"]
```

Comment:
I am just curious about why we set the entrypoint on `/tmp`. This fs should normally be mounted as `tmpfs` and hence be empty when the container starts. The fact that it just works is bugging me ....

Reply:
i used `/tmp` based on familiarity with *nix systems.  unsure what tradition `tmpfs` corresponds to.